pkg/cmd/search/shared/shared.go

```diff
@@ -95,7 +95,11 @@ func displayIssueResults(io *iostreams.IOStreams, et EntityType, results search.
 		if tp.IsTTY() {
 			issueNum = "#" + issueNum
 		}
-		tp.AddField(issueNum, nil, cs.ColorFromString(colorForIssueState(issue.State)))
```

Comment:
Should this use `pr/shared.ColorForIssueState()`?

Reply:
The search command has its own implementation of `colorForIssueState()`. At the time I was unsure how much overlap the output of `issue list` and `pr list` would have with `search issues` and `search prs` so I opted for duplicating some code instead of trying to keep it DRY.